app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+class ExpensesController < ApplicationController
+  # respond_to :json
+
+  def index
+    @budget_id = params[:budget_plan_id]
+    unless @budget_id.blank?
```

Comment:
nitpick: I find it more readable if you use `if @budget_id.present?`

Reply:
much more readable indeed